docs/_pages/releases.md

```diff
@@ -10,6 +10,7 @@ sidebar:
 ## Unreleased
 
 ### What's new
+* Added `ThatAreInterface` method for filtering the types - [#2057](https://github.com/fluentassertions/fluentassertions/pull/2057)
```

Comment:
You forgot to update the release notes...

Reply:
I was a bit overloaded with resolving the conflicts